Added a validation for integration image name (#1241)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
* Fixed an issue where in some cases the `get_remote_file` function failed due to an invalid path.

> * Added a validation that make sure the integration image name is a valid name.

Reviewer: ```suggestion
* Added a validation to ensure correct image file names.
```

Author: done.
